src/db/error.cr

```diff
@@ -3,6 +3,19 @@ module DB
   end
 
   class MappingException < Error
+    getter klass
```

Comment:
Is there a reason why you prefer the to store the class name instead of the metaclass directly?

Reply:
IMO it's good to keep exception types as simple es possible. Having the metaclass directly doesn't add a real benefit. It's just informational and a string should suffice. This makes it easier to introspect/export an exception instance.